source/components/form/form.ts

```diff
@@ -76,7 +78,7 @@ export class FormComponent {
 	}
 
 	validate(): boolean {
-		return this.form.valid;
+		return !this.form.invalid;
```

Comment:
Were you going to revert this since `pending` is actually another state that's not exactly valid?